### ISSUE TITLE
Requesting examples

### ISSUE THREAD

**Issue by clebio** [28mm/blast-radius-docs#1]
> Site Generation scripts not included in this distribution (request if desired).

Ok... requesting. :)

Less flippantly, have you thought about adding an `examples/` directory to the blast-radius repo itself? Any reason not to add some sample terraform code?

**Comment by nhoughto** [comment]
yeah first thing i thought when seeing blast radius was can I dump it to static html for it to become a 'terraform report' in CI. Be great to see how this can be achieved.
